notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,2380 @@
+{
```

Comment:
<p>In the other notebooks we have the first cell only with the imports and the version of key library of the notebook, I guess here it will be azureml. </p><p><br></p><p>For following this convention you would need to split the cell after the imports</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Fixed